scripts/subscribe-logs.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+/*
+Script will create a stream subscription between a log group
+and a lambda function (see arn, below).
+
+ENV vars for ACCOUNT_ID and REGION are required.
+*/
+
+// ACCOUNT_ID=xxxxxxxxx STAGE=omar npm run
+
+const AWS = require('aws-sdk');
+const cloudwatchlogs = new AWS.CloudWatchLogs({ region: 'us-east-1' });
```

Comment:
Should `us-east-1` be in an ENV var?

Reply:
+1 for env-var, I like using `eu-west-2` for my stack

Reply:
👍